bluecellulab/cell/core.py

```diff
@@ -233,44 +198,12 @@ def re_init_rng(self, use_random123_stochkv: bool = False) -> None:
             else:
                 self.cell.re_init_rng()
 
-    def get_psection(self, section_id=None, secname=None):
-        """Return a python section with the specified section id or name.
-
-        Parameters
-        ----------
-        section_id: int
-                    Return the PSection object based on section id
-        secname: string
-                 Return the PSection object based on section name
-
-        Returns
-        -------
-        psection: PSection
-                  PSection object of the specified section id or name
-        """
+    def get_psection(self, section_id: int | None = None) -> PSection:
+        """Return a python section with the specified section id or name."""
```

Comment:
How do I get the section id from a section object obtained with for example 
```
sec5 = cell.all[5]
sec5
```
The id is not available.

I've tried to pass the name of the section to this function but it does not work
```
cell.get_psection(sec5.hname())
# In the `self.psections` mapping it contains integers as keys
```

Reply:
There is the `cell.secname_to_psection` mapping. 
```python
secname_to_psection: dict[str, PSection] = {}
```
a) Can that be used to solve the hname to section mapping?
b) Would you like to have it as a method e.g. `get_psection_by_name`?

Reply:
Yeah I can use that. I'm just pointing out that in the docs it says something that is not true
```
"""Return a python section with the specified section id or name."""
```
Actually only ID works.

The original implementation it was different than the new one.
```
get_psection(self, section_id=None, secname=None)
```

Maybe we should do the internal magic of using `secname_to_psection` in the function if the name is passed?

Reply:
Oh, thanks for noticing the mismatch in the docs. 
> Maybe we should do the internal magic of using secname_to_psection in the function if the name is passed?

This sounds like a good idea. Let me update it.